packages/dx-react-scheduler/docs/reference/day-view.md

```diff
@@ -136,6 +136,9 @@ Field | Type | Description
 ------|------|------------
 startDate? | Date | Specifies the cell a start time.
 endDate? | Date | Specifies the cell end time.
```

Comment:
Do this mean that 'cell end time' is what it is called or do you mean 'the cell's end time'?

Reply:
```suggestion
endDate? | Date | Specifies the cell's end time.
```
@AryamnovEugeniy Same request here.